CONTRIBUTING.md

```diff
@@ -24,15 +29,15 @@ To build the PDK, run the following command:
 
 > These build scripts produce a binary for the current platform only. These scripts are meant for local testing. We use a Github Action to release this product.
 
-```bash
-> # on nix
+{{< tabpane langEqualsHeader=false >}}
```

Comment:
We can't have templating code in the markdown that's still being rendered in Github. We'll either need to have a separate `CONTRIBUTING` under `docs/md` or, given that is more developer focussed and the Hugo site is meant to be consumer focussed, we might not need to have that in `docs/md`

Reply:
I have removed the template code from the `CONTRIBUTING` file. For now the file will still be displayed on the documentation site, as I'll need to change the structure and module mount for the site on the `devx` repository to completely remove.